deploy/configmap.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: supported-nic-ids
+data:
+  I40e_XXV710: "8086 158a 154c"
```

Comment:
Shall we define a consistent name format for the data keys?
For example: 
<vendor_driver_model_name>
Intel_ice_E810_Columbiaville
Nvidia_mlx5_MT42882_ConnectX-5

Reply:
I think we can do that. But keep in mind, the NIC name here is not used in the code logic, it is just for readability.

Reply:
@pliurh are you going to update the keys here?

Reply:
done.

Reply:
done.